cnf-certification-test/chaostesting/pod_delete/pod_delete.go

```diff
@@ -216,17 +216,22 @@ func waitForResult() bool {
 		logrus.Errorf("error getting : %v\n", err)
 		return false
 	}
+
+	return parseLitmusResult(crs)
+}
+
+func parseLitmusResult(crs *unstructured.UnstructuredList) bool {
 	if len(crs.Items) > 1 {
-		logrus.Errorf("u have %d more than one chaosenging resource that is wrong bahavior \n", len(crs.Items))
+		logrus.Errorf("There are currently %d chaosengine resources. That is incorrect behavior \n", len(crs.Items))
```

Comment:
Space is not needed before the new line. Period at the end of the sentence will be nice.

Reply:
Updated.